app/views/devise/shared/_links.html.erb

```diff
@@ -20,6 +20,6 @@
 
 <%- if devise_mapping.omniauthable? %>
   <%- resource_class.omniauth_providers.each do |provider| %>
-    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), method: :post %><br />
+    <%= button_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), data: { turbo: false } %><br />
```

Comment:
Yeah, I think the best option for this one might be a button since this is not something that should go through turbo I guess?

The only other option that could work would be two methods. (`method: :post` and `data: { turbo_method: :post }`), but again I'm not sure about the Turbo behavior for something like this so may be better to disable it entirely.

Reply:
As far as I tested, Turbo didn't work with OAuth because it raises CORS error.

Here is the result when I click the link generated via `<%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), method: :post, data: { turbo_method: :post } %>`:

<img width="875" alt="Screenshot 2023-02-01 at 19 20 54" src="https://user-images.githubusercontent.com/1148320/216017551-4d790ef2-9683-4c2f-a02a-96f7448038ee.png">

So I had to change this line from link to button and add `data: { turbo: false }` option.

Reply:
Good point, I hadn't thought about CORS but yeah since it's all happening via JS it will trigger a CORS error when trying to follow the redirect with fetch under the hood.

`button_to` it is then, to support this correctly on all cases, since there's no way to do POST with turbo like rails-ujs does. Thanks!